service/src/test/java/uk/gov/hmcts/reform/fpl/service/validators/GroundsCheckerIsStartedTest.java

```diff
@@ -107,7 +108,7 @@ void shouldReturnFalseWhenGroundsForContactWithChildNotProvided() {
 
     private static Stream<Arguments> nonEmptyGrounds() {
         return Stream.of(
-                Grounds.builder().thresholdReason(List.of(("Test"))).build(),
+                Grounds.builder().groundsReason(List.of((GroundsList.NO_CARE))).build(),
```

Comment:
Think there's an extra pair of brackets on this one
```suggestion
                Grounds.builder().groundsReason(List.of(GroundsList.NO_CARE)).build(),
```